bin/run-browser-tests-local

```diff
@@ -0,0 +1,7 @@
+#! /usr/bin/env bash
+
+pushd browser-test
```

Comment:
Suggestions (non-blocking): this probably only works when run under main folder, consider using `$( dirname ${BASH_SOURCE[0]})/../browser-test` or something similar.

Reply:
Done.